pkg/frontend/frontend_series.go

```diff
@@ -4,13 +4,38 @@ import (
 	"context"
 
 	"github.com/bufbuild/connect-go"
+	"github.com/grafana/dskit/tenant"
+	"github.com/prometheus/common/model"
 
 	querierv1 "github.com/grafana/pyroscope/api/gen/proto/go/querier/v1"
 	"github.com/grafana/pyroscope/api/gen/proto/go/querier/v1/querierv1connect"
 	"github.com/grafana/pyroscope/pkg/util/connectgrpc"
+	"github.com/grafana/pyroscope/pkg/validation"
 )
 
 func (f *Frontend) Series(ctx context.Context, c *connect.Request[querierv1.SeriesRequest]) (*connect.Response[querierv1.SeriesResponse], error) {
 	ctx = connectgrpc.WithProcedure(ctx, querierv1connect.QuerierServiceSeriesProcedure)
+
+	tenantIDs, err := tenant.TenantIDs(ctx)
+	if err != nil {
+		return nil, connect.NewError(connect.CodeInvalidArgument, err)
+	}
+
+	if c.Msg.Start != 0 && c.Msg.End != 0 {
+		interval := model.Interval{
+			Start: model.TimeFromUnix(c.Msg.Start),
+			End:   model.TimeFromUnix(c.Msg.End),
```

Comment:
In order to be consistent with the rest of the api this needs to be milliseconds, something like that:

```suggestion
			Start: model.Time(int64(c.Msg.Start)),
			End: model.Time(int64(c.Msg.End)),
```

Reply:
I fixed all the timestamps in https://github.com/grafana/pyroscope/pull/2431/commits/47fca7040ae2f850364eaf1da82c06a7ad849630.